cloud/gcp/modules/compute/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "region" {
+  type        = string
+  description = "Default region for the db"
+}
+
+variable "connection_name" {
+    type    = string
+    description = "DB connection string"
```

Comment:
suggestion (blocking): if `connection_name` is specific to the DB please change its name to reflect that

Reply:
done